api/model.go

```diff
@@ -128,17 +128,22 @@ type NodeInput struct {
 
 type Node struct {
 	client.Resource
-	Name             string              `json:"name"`
-	Address          string              `json:"address"`
-	AllowScheduling  bool                `json:"allowScheduling"`
-	Disks            map[string]DiskInfo `json:"disks"`
-	State            types.NodeState     `json:"state"`
-	MountPropagation bool                `json:"mountPropagation"`
+	Name            string                                      `json:"name"`
+	Address         string                                      `json:"address"`
+	AllowScheduling bool                                        `json:"allowScheduling"`
+	Disks           map[string]DiskInfo                         `json:"disks"`
+	State           string                                      `json:"state"`
+	Conditions      map[types.NodeConditionType]types.Condition `json:"conditions"`
 }
 
 type DiskInfo struct {
 	types.DiskSpec
+	DiskStatus
+}
+
+type DiskStatus struct {
 	types.DiskStatus
+	State string `json:"state"`
```

Comment:
Expose the condition directly to the API. UI need to update API since there can be more than one condition set at the same time.